source/slang/slang-ir-reachability.cpp

```diff
@@ -54,6 +55,45 @@ namespace Slang
         }
 
     }
+    
+    bool ReachabilityContext::isInstReachable(IRInst* inst1, IRInst* inst2)
+    {
+        // If inst1 and inst2 are in the same block,
+        // we test if inst2 appears after inst1.
```

Comment:
If a certain order is expected for the input parameters, the naming may be better with `from` and `to`.
```
bool ReachabilityContext::isInstReachable(IRInst* from, IRInst* to)
```
Or `before` and `after`; or `top` and `down`... I don't know.